repository-data/application/src/main/resources/hcm-config/configuration/update/DW-366-GdprUpdateWhoCanAccessField.yaml

```diff
@@ -0,0 +1,15 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:update/hippo:registry/DW-366-GdprUpdateWhoCanAccessField:
+      hipposys:batchsize: 10
+      hipposys:description: ''
+      hipposys:dryrun: false
+      hipposys:loglevel: DEBUG
+      hipposys:parameters: '{}'
+      hipposys:query: /jcr:root/content/documents/corporate-website/about-nhs-digital/our-work/keeping-patient-data-safe/gdpr//*[@jcr:primaryType='hippo:handle']
```

Comment:
Just a minor comment here. You can query directly the variant nodes, by specifying the @jcr:primaryType property: I am sure that this folder (and subfolder) in production contains only gdprtransparency documents, but feel free to give a though and refactor it or not.